src/js/clipboard.js

```diff
@@ -0,0 +1,640 @@
+        return Object.prototype.hasOwnProperty.call(t, e);
+      }),
+      (e.p = ""),
+      e((e.s = 3))
+    );
+  })([
+    function (t, e, n) {
+      var o, r, i;
+      !(function (a, c) {
+        (r = [t, n(7)]),
+          (o = c),
+          void 0 !== (i = "function" == typeof o ? o.apply(e, r) : o) &&
+            (t.exports = i);
+      })(0, function (t, e) {
+        "use strict";
+        function n(t, e) {
+          if (!(t instanceof e))
+            throw new TypeError("Cannot call a class as a function");
+        }
+        var o = (function (t) {
+            return t && t.__esModule ? t : { default: t };
+          })(e),
+          r =
+            "function" == typeof Symbol && "symbol" == typeof Symbol.iterator
+              ? function (t) {
+                  return typeof t;
+                }
+              : function (t) {
+                  return t &&
+                    "function" == typeof Symbol &&
+                    t.constructor === Symbol &&
+                    t !== Symbol.prototype
+                    ? "symbol"
+                    : typeof t;
+                },
+          i = (function () {
+            function t(t, e) {
+              for (var n = 0; n < e.length; n++) {
+                var o = e[n];
+                (o.enumerable = o.enumerable || !1),
+                  (o.configurable = !0),
+                  "value" in o && (o.writable = !0),
+                  Object.defineProperty(t, o.key, o);
+              }
+            }
+            return function (e, n, o) {
+              return n && t(e.prototype, n), o && t(e, o), e;
+            };
+          })(),
+          a = (function () {
+            function t(e) {
+              n(this, t), this.resolveOptions(e), this.initSelection();
+            }
+            return (
+              i(t, [
+                {
+                  key: "resolveOptions",
+                  value: function () {
+                    var t =
+                      arguments.length > 0 && void 0 !== arguments[0]
+                        ? arguments[0]
+                        : {};
+                    (this.action = t.action),
+                      (this.container = t.container),
+                      (this.emitter = t.emitter),
+                      (this.target = t.target),
+                      (this.text = t.text),
+                      (this.trigger = t.trigger),
+                      (this.selectedText = "");
+                  },
+                },
+                {
+                  key: "initSelection",
+                  value: function () {
+                    this.text
+                      ? this.selectFake()
+                      : this.target && this.selectTarget();
+                  },
+                },
+                {
+                  key: "selectFake",
+                  value: function () {
+                    var t = this,
+                      e = "rtl" == document.documentElement.getAttribute("dir");
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)